src/components/Settings/AdvancedSettings.tsx

```diff
@@ -26,13 +26,19 @@ export default function TitleChanger() {
   };
 
   return (
-    <form onSubmit={form.onSubmit((values) => saveChanges(values))}>
-      <TextInput label="Page title" placeholder="'Homarr 🦞" {...form.getInputProps('title')} />
-      <TextInput label="Logo" placeholder="/img/logo.png" {...form.getInputProps('logo')} />
-      <TextInput label="Favicon" placeholder="/favicon.svg" {...form.getInputProps('favicon')} />
-      <Group grow position="center" mt="xl">
-        <Button type="submit">Save</Button>
-      </Group>
-    </form>
+    <Group direction="column" grow>
```

Comment:
What's the purpose of wrapping the form in a group? Commenting it out doesn't change anything, and you do the same thing in `CommonSettings.tsx` but you wrap everything in two groups, even though they're the  same.